web/browser.py

```diff
@@ -6,36 +6,18 @@
 from io import BytesIO
 
 from .net import htmlunquote
-from .py3helpers import PY2, text_type
 from .utils import re_compile
 
-try:  # Py3
-    from http.client import HTTPMessage
-    from urllib.request import HTTPHandler, HTTPCookieProcessor, Request, HTTPError
-    from urllib.request import build_opener as urllib_build_opener
-    from urllib.parse import urljoin
-    from http.cookiejar import CookieJar
-    from urllib.response import addinfourl
-except ImportError:  # Py2
-    from httplib import HTTPMessage
-    from urllib import addinfourl
-    from urllib2 import HTTPHandler, HTTPCookieProcessor, Request, HTTPError
-    from urllib2 import build_opener as urllib_build_opener
-    from cookielib import CookieJar
-    from urlparse import urljoin
-
-# Welcome to the Py2->Py3 httplib/urllib reorganization nightmare.
-
-if PY2:
-    get_selector = lambda x: x.get_selector()
-    get_host = lambda x: x.get_host()
-    get_data = lambda x: x.get_data()
-    get_type = lambda x: x.get_type()
-else:
-    get_selector = lambda x: x.selector
-    get_host = lambda x: x.host
-    get_data = lambda x: x.data
-    get_type = lambda x: x.type
+from urllib.request import HTTPHandler, HTTPCookieProcessor, Request, HTTPError
+from urllib.request import build_opener as urllib_build_opener
+from urllib.parse import urljoin
+from http.cookiejar import CookieJar
+from urllib.response import addinfourl
+
+get_selector = lambda x: x.selector
+get_host = lambda x: x.host
+get_data = lambda x: x.data
+get_type = lambda x: x.type
```

Comment:
These seem like a lot of noise for little value.  Can they be removed?

Reply:
Next PR.